internal/cloudinit/cloudinit.go

```diff
@@ -97,20 +97,20 @@ swapon /swapfile
 echo "done setting up swap space"
 
 echo "downloading lite engine binary"
-/usr/bin/wget --retry-connrefused --retry-on-host-error --retry-on-http-error=503,404,429 --tries=10 --waitretry=10 "{{ .LiteEnginePath }}/lite-engine-{{ .Platform.OS }}-{{ .Platform.Arch }}" -O /usr/bin/lite-engine
+/usr/bin/wget --retry-connrefused --retry-on-host-error --retry-on-http-error=503,404,429 --tries=10 --waitretry=10 "{{ .LiteEnginePath }}/lite-engine-{{ .Platform.OS }}-{{ .Platform.Arch }}" -O /usr/bin/lite-engine || /usr/bin/wget --retry-connrefused --tries=10 --waitretry=10 -nv --debug "{{ .LiteEnginePath }}/lite-engine-{{ .Platform.OS }}-{{ .Platform.Arch }}" -O /usr/bin/lite-engine
```

Comment:
let's refactor it a bit, put ` {{ .LiteEnginePath }}/lite-engine-{{ .Platform.OS }}-{{ .Platform.Arch }}" -O /usr/bin/lite-engine` in variable and then do `wget --retry-connrefused --retry-on-host-error --retry-on-http-error=503,404,429 --tries=10 --waitretry=10 var1`
Basically clean it up a little at all the places

Reply:
The reason we are doing it this way because we are not doing only for LE there are plugins, splitTests and the path 
"/usr/bin/lite-engine" is different for some other LE binary downloads commands, same for plugin as well. So ultimately we will end up with many such var strings

Reply:
We will have one such variable per wherever we are using this wget || thing and that's alright I guess, as long as it helps removing duplicate code.